native/package.json

```diff
@@ -50,10 +50,10 @@
     "@notifee/react-native": "^7.8.2",
     "@react-native-async-storage/async-storage": "^1.23.1",
     "@react-native-clipboard/clipboard": "^1.14.0",
-    "@react-native-community/geolocation": "^3.2.1",
+    "@react-native-community/geolocation": "3.0.6",
```

Comment:
I had to downgrade because of this (or a similar) error:
https://github.com/rnmapbox/maps/issues/3046